graspy/match/gmp.py

```diff
@@ -57,6 +57,19 @@ class GraphMatch(BaseEstimator):
         Gives users the option to solve QAP rather than the Graph Matching Problem
         (GMP). This is accomplished through trivial negation of the objective function.
 
+    padding : string (default = 'adopted')
+        Allows user to specify padding scheme if `A` and `B` are not of equal size.
+
+        "adopted" : say that `A` and `B` have :math:`n_1` and :math:`n_2` nodes, respectively.
+        Define :math:`\widetilde{A} = 2A - 1_{n1}1_{n1}^T` and
+        :math:`\widetilde{B} = 2B - 1_{n2}1_{n2}^T`. If :math:`n_1 < n_2`, then we substitute
+        :math:`\widetilde{A} \oplus 0_{(n2-n1)x(n2-n1)}` and :math:`\widetilde{B}` in place of A
+        and B.
+
+        "naive" : say that `A` and `B` have :math:`n_1` and :math:`n_2` nodes, respectively.
+        If :math:`n_1 < n_2`, then we substitute :math:`A \oplus 0_{(n2-n1)x(n2-n1)}` and
+        :math:`B` in place of A and B.
```

Comment:
i don't know if this is the appropriate place for the math or not. regardless, none of this helps the user understand what to use and when

Reply:
Makes sense. I'll add this math part to the tutorial.